Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/controller/SuperAdminController.java

```diff
@@ -2,14 +2,8 @@
 
 import com.hydraulic.applyforme.model.domain.Member;
 import com.hydraulic.applyforme.service.SuperAdminService;
-import com.hydraulic.applyforme.service.impl.SuperAdminServiceImpl;
-import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping(
```

Comment:
Since this is admin controller. We can set the value on @RequestMapping to "admin"